app/templates/reminders/new.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="add-reminder">
+
```

Comment:
Let's clean up the extra lines here - also in general I'd use CSS to add spacing between input fields, not `<br/>` tags. It looks like you already have a css class on them so it wouldn't take long to add a bottom margin.